astropy/coordinates/tests/test_representation.py

```diff
@@ -1046,6 +1049,16 @@ class LogDRepresentation(BaseRepresentation):
                                         ('lat', Latitude),
                                         ('logr', u.Dex)])
 
+    # now try passing in a differential
+    diff = CartesianDifferential(d_x=1 * u.km/u.s,
```

Comment:
Yes, agreed to postpone. Though I'm not sure I agree with your statement about the unit vectors and scale factor; log-distance is a strange space. In fact, perhaps not the best example for the tests or docs...but let's revisit as a "doc change"